_devExtensions/api.js

```diff
@@ -15,6 +15,21 @@ const getIp = {
   }
 }
 
```

Comment:
Hello @TahaKhanAbdalli - when I was implementing this new feature I thought I was re-using the existing `validateCurrentPassword` method.  

I now realise that Github CoPilot write it for me.  I'm pretty amazed at that. It all looks valid to me, and works correctly - but I'd appreciate you giving it extra scrutiny please.